app.js

```diff
@@ -0,0 +1,64 @@
+const template = document.createElement('template');
+const form = document.forms['books-form'];
+const oldBooks = JSON.parse(localStorage.getItem('booksArray')) || [];
+
+function grab(e) {
+  return document.getElementById(e);
+}
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  deleteBook(index) {
+    oldBooks.splice(index, 1);
+    localStorage.setItem('booksArray', JSON.stringify(oldBooks));
+    grab('books-container').innerHTML = '';
+    initialLoad();
+  }
+
+  addBook() {
+    const title = form.booktitle;
+    const author = form.bookauthor;
+    const bookItem = new Book(title.value, author.value);
+    oldBooks.push(bookItem);
+
+    localStorage.setItem('booksArray', JSON.stringify(oldBooks));
+    document.forms['books-form'].reset();
+    grab('books-container').innerHTML = '';
+    initialLoad();
+  }
+}
+const bookf = new Book();
+function initialLoad() {
```

Comment:
- [ ] I would recommend using the arrow syntax for the function. As it is requirement to use ES6 syntax as much as you can to get familiar with.

Reply:
Thank you for the feedback. However the instruction stated that I should create the index.js file and the modules and those should follow es6 syntax. Some browsers do not support Es6 syntax so that is where the app.js file comes in. It works in the unsupported browser if you take a look in the index.html file you'll see that I added a nomodule tag to the script tag used to import the app.js. Please have another look